app/mailers/notification_mailer.rb

```diff
@@ -11,6 +11,12 @@ def daily_summary_email(addressee, summary_report = nil)
     mail(to: addressee.email, subject: t('.subject', date: Date.yesterday))
   end
 
+  def daily_auctions_broadcast_email(addressee, auctions)
```

Comment:
How about using keyword arguments?

Reply:
What's the benefit of keyword arguments in this scenario?